diff -urN aaa/dopewars-1.5.9/dopewars.desktop dopewars-1.5.9/dopewars.desktop
--- aaa/dopewars-1.5.9/dopewars.desktop	2001-05-30 04:03:44.000000000 +0200
+++ dopewars-1.5.9/dopewars.desktop	2003-12-10 16:27:13.000000000 +0100
@@ -3,7 +3,7 @@
 Name[de]=Drogenkrieg
 Name[pl]=Wojny narkotykowe
 Comment=dopewars drug dealing game
-Comment[pl]=Gra polegajÄca na handlowaniu narkotykami
+Comment[pl]=Gra polegaj±ca na handlowaniu narkotykami
 Comment[fr]=Jeu de vente de drogue
 Exec=dopewars
 Icon=dopewars-weed.png
